rllib/evaluation/collectors/agent_collector.py

```diff
@@ -189,17 +189,38 @@ def add_init_obs(
         # view requirement does not match init_obs
         self._check_view_requirement(SampleBatch.OBS, init_obs)
 
+        if SampleBatch.PREV_REWARDS in self.view_requirements:
+            prev_reward = get_dummy_batch_for_space(
+                    space=self.view_requirements[SampleBatch.REWARDS].space,
+                    batch_size=1,
+                    fill_value=.0
+                )[0]
+        if SampleBatch.PREV_REWARDS in self.view_requirements:
+            prev_action = get_dummy_batch_for_space(
+                space=self.view_requirements[SampleBatch.ACTIONS].space,
+                batch_size=1,
+                fill_value=.0
+            )[0]
```

Comment:
why not just inline the get_dummy_batch_for_space() calls with the if blocks below?

Reply:
Artifact of a failed solution, changed, thanks!